src/main/java/com/thealgorithms/maths/FibCalc.java

```diff
@@ -0,0 +1,17 @@
+import java.math.BigInteger;
+
+public class FibCalc {
+    public static BigInteger calFib(int n) {
+        if (n <= 1) {
```

Comment:
What if `n < 0`? Please throw `IllegalArgumentException` and add a corresponding test.